modules/controlfreec/CHANGELOG.md

```diff
@@ -38,4 +38,22 @@ This release was authored by Jasper.
 
 Updated controlfreec.smk to allow for running on Numbers cluster.
 
-Fixed bugs related to genome builds and preparation of controlfreec-specific reference files. Now will generate chr.len file for hg19, hg38 (w/ chr prefix), grch37, grch38 (no chr).
\ No newline at end of file
+Fixed bugs related to genome builds and preparation of controlfreec-specific reference files. Now will generate chr.len file for hg19, hg38 (w/ chr prefix), grch37, grch38 (no chr).
+
+## [1.2] - 2020-09-10
+
+This release was authored by Jasper.
+
+Fixed a config bug (snakemake replacing 3&4 with 3numdegree4 in config_WGS.txt) - added \ on top of &. Note: tested it, the output was the same (since 3&4 is default)
+Changed output dir to be controlfreec-1.1/ instead of controlfreec-1.0/
+Added a new feature in snakemake to allow for conversion of CNV calls to bed.
+
+## [1.2] - 2020-11-05
+
+This release was authored by Jasper.
+
+Improved control-freec calls for FFPE genomes. Everything is now default in paired mode with BAF turned on to measure LOH/BAF. All unmatched samples will be ran with a default normal. This will now generate temporary pileups in a scratch space (time consuming). The parameters have been set to minimize noise while also maximizing the number of prominent signals across the genome. To accomplish this, I set the default ploidy to 2 to prevent control-freec from overfitting the model for unpaired samples when ploidy != 2. Importantly, I set the parameter forceGCcontentNormalization=1, which forces GC normalzation for both the sample and control read counts (RC), and then calculates the ratio of sample RC/control RC. Then I changed the intercept of the polynomial from 1 to 0 to make the valuation of sample relative to the control instead of to its own GC content. This effectively smoothed out the profiles without added segments of overamplified "noise". I also allowed control-freec to assess the relative normal contamination and allowed control-freec to set its own window size given the coverage across the genome. These parameters effectively smoothed out unnecessary noise even further.
```

Comment:
`I set the default ploidy to 2 to prevent control-freec from overfitting the model for unpaired samples when ploidy != 2`
But it still detects CNVs, right? What is the rough range of ploidy for detected CNVs?

Reply:
Yes, it detects all CNVs relative to ploidy = 2. Before, when I set the range to 2,3,4, it seems like there would be some over-fitting done, where certain samples would be assigned something like ploidy = 3 and then show every chromosome as a CN loss. It felt like, especially with unpaired samples, that setting a default to 2 would give the most meaningful results

Reply:
Again, it would be good to leave a short comment in the default config that indicates a range can be provided